http/http.go

```diff
@@ -93,20 +95,27 @@ func (s *Server) newResponse(r *http.Request) (Response, error) {
 	ipDecimal := iputil.ToDecimal(ip)
 	country, _ := s.gr.Country(ip)
 	city, _ := s.gr.City(ip)
+	asn, _ := s.gr.ASN(ip)
 	var hostname string
 	if s.LookupAddr != nil {
 		hostname, _ = s.LookupAddr(ip)
 	}
+	var autonomousSystemNumber string
+	if asn.AutonomousSystemNumber > 0 {
+		autonomousSystemNumber = "AS" + strconv.FormatUint(uint64(asn.AutonomousSystemNumber), 10)
```

Comment:
Any particular reason for the uint64 cast? If not, use `strconv.Itoa`.

Reply:
I'm not sure. This, like most of the code, is stolen directly from the original PR.
Let me expand: is there are particular reason why this should be an uint64? Up until that point it's an uint, as mandated by [IANA](https://tools.ietf.org/html/rfc6996).

cc @raunsbaekdk